annotationsx/settings/base.py

```diff
@@ -269,3 +302,12 @@
     pass
 elif ORGANIZATION == "HARVARDX":
     pass
+
+
+PYLTI_CONFIG = {
```

Comment:
Question on this: I'm wondering if this settings dict is needed given that `lti_validators.py` is pulling the consumer key and secret from the settings module directly (e.g. `settings.CONSUMER_KEY` and `settings.LTI_SECRET`). Do we know where/how this is being used? Or is it possible to remove it?

Reply:
Yeah you're right, I think it can be removed. Thanks for the head's up!